code/modules/emoji/emoji_parse.dm

```diff
@@ -8,11 +8,11 @@
 	var/search = 0
 	var/emoji = ""
 	while(1)
-		search = findtext(text, ":", pos)
+		search = findtext(text, ";", pos)
 		parsed += copytext(text, pos, search)
 		if(search)
 			pos = search
-			search = findtext(text, ":", pos+1)
+			search = findtext(text, ";", pos+1)
```

Comment:
Instead of trying to resolve conflicts this way, I think we should allow our custom emoji to override twemoji if there are any clashes, and if you want to fix clashes just change the names of our emoji.

Doing this will likely make emoji.dmi unused and fall away.